### DISCUSSION TITLE
Generate copyWith method without proxy.

### DISCUSSION THREAD

**Discussion by p4-k4** [numen31337/copy_with_extension#51] [General]
Is there currently a way to generate a raw form of a `copyWith` method without the whole proxy business?

For example:
```dart
// From this.
class Person {
  const Person({required this.firstName, required this.lastName});
  final String firstName;
  final String lastName;
}

// To just generating this.
copyWith(String firstName, String lastName) {
  return Person(firstName: firstName, lastName: lastName);
}
```

Thanks.

**Comment by numen31337** [comment]
Unfortunately, no. You can try to fork some previous version where it was generating just and extension with this function. As for now it is replaced with this proxy class which is more superior in terms of number of use cases it covers. Are there any particular problems with the current approach that are avoidable by having a plain extension `copyWith` function?
